test/ci.test.js

```diff
@@ -41,3 +41,18 @@ test("is CI required false", async () => {
   // Assert
   expect(!result);
 });
+
+test("isCIRequired paths file not defined", async () => {
+  //Arrange
+  const context = {
+    config: jest.fn(path => (path === "bot-files/paths.yml" ? null : undefined))
```

Comment:
and here I would replace this by either
`config: jest.fn(path => (path === "bot-files/paths.yml" ? undefined : {}))` or `config: jest.fn(() => undefined)`
first way you assure the `context.config` function will be always called with `bot-files/paths.yml`, second case you don't care about parameter (since the case is already covered by other test methods) and you just always return `undefined`.
Choose the one you feel more comfortable with :wink:

Reply:
Ah yes, the parameter does not matter, and that context.config returns null in case file is not defined, so updated tests accordingly.
Thanks for the suggestions :)